website_google_tag_manager/views/res_config_settings_view.xml

```diff
@@ -8,7 +8,7 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="website.res_config_settings_view_form" />
         <field name="arch" type="xml">
-            <div id="plausbile_setting" position="after">
+            <setting id="plausbile_setting" position="after">
                 <div
                     class="col-xs-12 col-lg-6 o_setting_box"
```

Comment:
Please change this according to the migration guide to `<setting>` and company:

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

Reply:
Changed according to the migration guide. Please review.